### PR TITLE
Present exposure stats through the Vulcan API

### PR DESCRIPTION
**Pull request by gguillemas**
This PR allows users to access the exposure stats endpoints [incorporated in the Vulnerability DB API](https://github.com/adevinta/vulnerability-db-api/pull/17) through the Vulcan API. The endpoints have been exposed both for specific teams and globally almost identically as with the MTTR stats endpoints. The changes have been tested with the linked PR and work as expected.

### CODE REVIEW

File: pkg/vulnerabilitydb/vulnerabilitydb_test.go [hunk 1]
```diff
@@ -595,6 +606,237 @@ func TestStatsMTTR(t *testing.T) {
...
+				t.Fatalf("No error expected, but got '%v'", err)
+			} else {
+				t.Fatalf("Expecting error '%v', but got '%v'", tt.expectedErr, err)
+			}
+		}
+		if err == nil && !reflect.DeepEqual(resp, mockStatsExposure) {
+			t.Errorf("Expected response body as:\n%v\nBut got:\n%v", mockStatsExposure, resp)
+		}
+
+		server.Close()
```
Comment (filewalkwithme) [inline pkg/vulnerabilitydb/vulnerabilitydb_test.go:836]: Can we move the `Close()` method to a `defer` instruction placed just after the server instantiation?

File: cmd/vulcan-api-cli/design/stats.go [hunk 2]
```diff
@@ -31,6 +31,27 @@ var StatsTotalMedia = MediaType("statsTotal", func() {
 	})
 })
 
+var StatsAveragesMedia = MediaType("statsAverages", func() {
```
Comment (filewalkwithme) [inline cmd/vulcan-api-cli/design/stats.go:34]: This `statsAverages` seems to be unused, is that correct?

**Review by filewalkwithme — CHANGES_REQUESTED** [review]
Hi @gguillemas, I have added just 2 minor comments. Let me know if you have any questions.

File: cmd/vulcan-api-cli/design/stats.go (hunk 2 above)
Comment (gguillemas) [reply cmd/vulcan-api-cli/design/stats.go:34]: You are correct, I forgot to replace the media type used on the endpoint. Fixed in d7d856f.

File: pkg/vulnerabilitydb/vulnerabilitydb_test.go (hunk 1 above)
Comment (gguillemas) [reply pkg/vulnerabilitydb/vulnerabilitydb_test.go:836]: As we discussed directly, this line is copied from the other tests and should be addressed for all in a separate PR.

**Review by filewalkwithme — APPROVED** [review]
LGTM

File: pkg/vulnerabilitydb/vulnerabilitydb_test.go (hunk 1 above)
Comment (filewalkwithme) [reply pkg/vulnerabilitydb/vulnerabilitydb_test.go:836]: Agree!

**Pull request merged**